enterprise/views.py

```diff
@@ -1023,6 +1023,10 @@ def get(self, request):
             # Add the tpa_hint to the redirect's query parameters
             tpa_hint = enterprise_customer.identity_provider
             query_dict['tpa_hint'] = tpa_hint
+        else:
+            # Add Enterprise customer UUID and proxy_login to the redirect's query parameters
+            query_dict['enterprise_customer'] = [str(enterprise_customer.uuid)]
+            query_dict['proxy_login'] = [True]
```

Comment:
Out of curiosity, do the values (e.g., `str(enterprise_customer.uuid)`) need to wrapped in a list?

Reply:
this was a linter workaround I made because parse_qs (line 1012) returns a Dict[AnyStr, List[AnyStr]] . The linter didn't complain about assigning tpa_hint or next_param because they have the type Any